synthesizer/event.cpp

```diff
@@ -180,6 +181,15 @@ bool NPlayEvent::isMuted() const
             const Channel* a = instr->playbackChannel(n->subchannel(), cs);
             return a->mute() || a->soloMute() || !staff->playbackVoice(n->voice());
             }
+      else if (score) {
```

Comment:
This is something that I think would be a great idea. It would also really help with the harmony channels issue since it'll let us have finer control over all channels across time.